catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/BrickAdapter.kt

```diff
@@ -389,10 +390,24 @@ class BrickAdapter(private val sprite: Sprite) :
             } else {
                 destinationList.add(itemToMove)
             }
+            checkAndSetCommentedOut(brickAboveTargetPosition, itemToMove)
         }
         updateItemsFromCurrentScripts()
     }
 
+    private fun checkAndSetCommentedOut(brickAboveTargetPosition: Brick, itemToMove: Brick?) {
+        if (brickAboveTargetPosition.script != null) {
+            if (brickAboveTargetPosition.script.isCommentedOut) {
+                itemToMove?.isCommentedOut = true
+            } else if (brickAboveTargetPosition is CompositeBrick && brickAboveTargetPosition.isCommentedOut) {
```

Comment:
This way every block added below (but outside) the end of a disabled loop is also disabled, is this intended behaviour?

Reply:
LGTM now!